README.md

```diff
@@ -2,12 +2,15 @@
 
 Scenarioo Writer API for Java to generate Scenarioo Documentations - http://www.scenarioo.org
 
-For documentation please refer to the [wiki](https://github.com/scenarioo/scenarioo-java/wiki).
 
-## Build the Java Writer API
+## Usage
+Scenarioo-Java is available on maven central: Include the latest version as a dependency in your favorite 
```

Comment:
For avoiding duplication I would only link to the usage guide document here and only explain there (with a maven example) how to link the library exactly - currently the usage guide still mentions our own maven repo.